karma.conf.js

```diff
@@ -0,0 +1,23 @@
+module.exports = config => {
+    config.set({
```

Comment:
Karma replaces Cypress as the unit test runner. Configuration to run in the desired env the desired tests correctly bundled.